lexicons/community/atprotocol/map.json

```diff
@@ -0,0 +1,41 @@
+{
+  "lexicon": 1,
+  "id": "community.atprotocol.geomarker.map",
+  "defs": {
+    "main": {
+      "type": "record",
+      "key": "tid",
+      "record": {
+        "type": "object",
+        "required": ["locations"],
+        "properties": {
+          "label": {
+            "type": "string",
+            "minLength": 1,
+            "maxLength": 800,
+            "maxGraphemes": 400
+          },
+          "description": {
+            "type": "string",
+            "minLength": 1,
+            "maxLength": 600,
```

Comment:
I would make this length at least 6000. personally. Also this is usually smaller than `maxGraphemes`.